packages/web/app/tsconfig.json

```diff
@@ -25,6 +25,6 @@
       }
     ]
   },
-  "include": ["src"],
+  "include": ["src", "vite.config.ts", "tailwind.config.ts"],
```

Comment:
```suggestion
  "include": ["**/*.ts", "**/*.tsx", "next-env.d.ts", ".next/types/**/*.ts"],
```
next.js default

Reply:
good idea

Reply:
Reverted the suggestion because Hive app is a Next.js app, and there's also a `dist` there that should be added to excludes if we want to include all .tsx